src/software/ai/hl/stp/tactic/dribble/dribble_tactic_test.cpp

```diff
@@ -315,3 +315,37 @@ TEST_F(DribbleTacticTest, test_running_into_enemy_robot_knocking_ball_away)
             terminating_validation_functions, non_terminating_validation_functions,
             Duration::fromSeconds(20));
 }
+
+TEST_F(DribbleTacticTest, test_robot_not_bumping_ball_when_turning_around)
```

Comment:
consider writing a pytest for this test (leave the C++ test so that you don't have to write all the tests)